src/Button/Button.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component, PropTypes} from 'react';
+import propTypes from './buttonPropTypes';
+import EnhancedButton from '../internal/EnhancedButton';
+import getStyles from './buttonStyles';
+import styleButtonChildren from './styleButtonChildren';
+
+import {
+  handleMouseDown,
+  handleMouseUp,
+  handleMouseLeave,
+  handleMouseEnter,
+  handleTouchStart,
+  handleTouchEnd,
+  handleKeyboardFocus,
+} from './buttonEventHandlers';
+
+class Button extends Component {
+  static muiName = 'Button';
+  static propTypes = propTypes;
+
+  static defaultProps = {
+    disabled: false,
+    fullWidth: false,
+    onKeyboardFocus: () => {},
+    onMouseDown: () => {},
+    onMouseEnter: () => {},
+    onMouseLeave: () => {},
+    onMouseUp: () => {},
+    onTouchEnd: () => {},
+    onTouchStart: () => {},
+    primary: false,
+    secondary: false,
+    type: 'raised',
```

Comment:
Quick Q for both of you @mbrookes @oliviertassinari 

It seems like we're embracing two different techniques for theme/color and type. We're using booleans for primary/secondary yet a string enum for type.

Or am I going crazy?